### PR TITLE
[jormungandr]: not more than 3 successive buses allowed in secnario STIF

### PR DESCRIPTION
**Pull request by kadhikari**
Ticket concerned : http://jira.canaltp.fr/browse/NAVP-573

### CODE REVIEW

File: source/jormungandr/jormungandr/scenarios/journey_filter.py [hunk 1]
```diff
@@ -201,6 +203,34 @@ def _filter_too_long_waiting(journeys, request):
             mark_as_dead(j, "too_long_waiting")
             break
 
+
+def _filter_max_successive_buses(journeys, request):
+    """
+    eliminates journeys with public_transport.bus more than _max_successive_buses
+    """
+    logger = logging.getLogger(__name__)
+    min_successive_buses = get_or_default(request, '_max_successive_buses', 0)
```
Comment (antoine-de) [inline source/jormungandr/jormungandr/scenarios/journey_filter.py]: it's max_successsive_buses

File: source/jormungandr/jormungandr/scenarios/journey_filter.py [hunk 2]
```diff
@@ -201,6 +203,34 @@ def _filter_too_long_waiting(journeys, request):
...
+    if (min_successive_buses == 0):
```
Comment (antoine-de) [inline source/jormungandr/jormungandr/scenarios/journey_filter.py]: you don't need the `(` `)`

File: source/jormungandr/tests/stif_tests.py [hunk 3]
```diff
@@ -85,3 +86,61 @@ def test_stif_override_final_line_filter(self):
         eq_(response['journeys'][0]['arrival_date_time'], '20140614T100000')
         eq_(response['journeys'][1]['arrival_date_time'], '20140614T110000')
         eq_(response['journeys'][2]['arrival_date_time'], '20140614T120000')
+
+    def test_stif_max_successive_buses(self):
```
Comment (antoine-de) [inline source/jormungandr/tests/stif_tests.py:90]: great tests :+1:

**Comment by kadhikari** [comment]
@antoine-de 
I made some corrections on your comments

**Pull request merged**